core/deployment/src/main/java/io/quarkus/deployment/pkg/builditem/DockerImageResultBuildItem.java

```diff
@@ -0,0 +1,30 @@
+
+package io.quarkus.deployment.pkg.builditem;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class DockerImageResultBuildItem extends SimpleBuildItem {
+
+    private final String imageId;
+    private final String repository;
+    private final String tag;
```

Comment:
Do we need these fields? They seem to be initialised to `null` all the time 
- https://github.com/quarkusio/quarkus/pull/5496/files#diff-807046a1a852a9c1d26116f35cf9ef16R35
- https://github.com/quarkusio/quarkus/pull/5496/files#diff-807046a1a852a9c1d26116f35cf9ef16R51

Reply:
Same as above.